pulp_smash/tests/rpm/api_v2/test_repomd.py

```diff
@@ -0,0 +1,60 @@
+# coding=utf-8
+"""Test repomd repository generated by yum distributor."""
+from __future__ import unicode_literals
+
+from pulp_smash import api
+from pulp_smash.compat import urljoin
+from pulp_smash.constants import REPOSITORY_PATH
+from pulp_smash.tests.rpm.api_v2.utils import gen_repo, gen_distributor, \
+    BaseRepoMDTestCase, xml_handler
+
+
+class RepoMDTestCase(BaseRepoMDTestCase):
```

Comment:
Is there any special reason to inherit from `BaseRepoMDTestCase`? As far as I can tell, this class uses nothing from that parent, and it can inherit from `BaseAPITestCase` instead.